packages/controller-utils/src/types.ts

```diff
@@ -30,19 +30,38 @@ export function isNetworkType(val: any): val is NetworkType {
   return Object.values(NetworkType).includes(val);
 }
 
-export enum NetworksChainId {
-  mainnet = '1',
-  goerli = '5',
-  sepolia = '11155111',
-  aurora = '1313161554',
-  rpc = '',
+/**
+ * Names of networks built into the wallet.
+ *
+ * This includes both Infura and non-Infura networks.
+ */
+export enum BuiltInNetwork {
```

Comment:
Nit: Should this be BuiltInNetworkName instead of BuiltInNetwork? I'm curious whether it is worth leaving the possibility open for making a [composite enum for networks like the extension has](https://github.com/MetaMask/metamask-extension/blob/develop/shared/constants/network.ts#L274) (which we could presumably call BuiltInNetwork). I also realize that referring to the name of a network as just "network" is pretty common here, in the extension, and also in https://github.com/MetaMask/eth-json-rpc-infura, so I'm fine either way. Just a thought.

Reply:
Good idea; better to change this now before this is used anywhere else

Reply:
Done in https://github.com/MetaMask/core/pull/1354/commits/a683eb2cf55fc0ccc0bd88d2c4e3afd737e1f4aa